AdminWebsite/AdminWebsite/Models/HearingRole.cs

```diff
@@ -1,20 +1,8 @@
-using System;
-
 namespace AdminWebsite.Models
 {
-    public class HearingRole : IComparable
+    public class HearingRole
     {
         public string Name { get; set; }
         public string UserRole { get; set; }
-        
-        public int CompareTo(object? obj)
```

Comment:
Keep this. We use this for ordering hearing roles by name

Reply:
but there are no usages, and caused a code smell?

Reply:
@shaed-parkar ?

Reply:
@shaed-parkar I have implemented it properly now its used. It needs all the operands otherwise it causes a code smell.

Reply:
Resolving as I have added it in and fixed the code smell